### PR TITLE
adds liches and phylacteries

### PR DESCRIPTION
**Pull request by MadmanMartian**
Sprites and ideas courtesy of @Blithering

### CODE REVIEW

**Comment by spoonylove** [comment]
Wew. Hiding phylacteries in the dorm toilets and the clowns ass when??

File: code/game/gamemodes/wizard/artefact.dm [hunk 1]
```diff
@@ -198,4 +198,74 @@
...
+		else
+			icon_state = "phylactery_empty"
+	else
+		icon_state = "phylactery_empty_noglow"
+
+/obj/item/phylactery/attack_self(mob/user)
+	if(!soulbound && ishuman(user))
+		var/mob/living/carbon/human/H = user
+		var/datum/organ/external/E = H.get_active_hand_organ()
+		if(locate(/datum/wound) in E.wounds)
```
Comment (unid15) [inline code/game/gamemodes/wizard/artefact.dm:253]: eh feel free go ignore this tiny little thing, but it'd be better if this was an organ-level proc, for instance ``if(E.is_wounded())``

**Comment by SonixApache** [comment]
when

**Comment by MadmanMartian** [comment]
mother hubbard

**Comment by DamianX** [comment]
fix it

**Comment by MadmanMartian** [comment]
[scream]

**Comment by MadmanMartian** [comment]
its done

**Pull request merged**